focus-areas/contributions/types-of-contributions.md

```diff
@@ -1,82 +1,67 @@
-* Account for activity from bots when mining contribution types at large scale.
-
-## References
-* https://medium.com/@sunnydeveloper/revisiting-the-word-recognition-in-foss-and-the-dream-of-open-credentials-d15385d49447
-* https://24pullrequests.com/contributing
-* https://smartbear.com/blog/test-and-monitor/14-ways-to-contribute-to-open-source-without-being/
-* https://wiki.openstack.org/wiki/AUCRecognition
-* https://www.drupal.org/drupalorg/blog/a-guide-to-issue-credits-and-the-drupal.org-marketplace
-
-## Contributors  
-- Kevin Lumbard  
-- Vinod Ahuja  
-- Elizabeth Barron  
-- Dawn Foster  
-- Sean Goggins  
-- Matt Germonprez  
+# **Types of Contributions**
+
+**Question:** What types of contributions are being made?
+
+## **Overview**
+Multiple, varied contributions make an open-source project healthy. Contributions may include writing code, managing the community, triaging bugs, evangelizing the project, supporting users, or helping in other ways.  
+A variety of contribution types demonstrate that a project is mature and well-rounded with sufficient activity to support all aspects of the project, and enable paths to leadership that are supportive of various expertise beyond coding.  
+Additionally, types of contributions along with project demographic information may provide insight into the diversity and inclusiveness of a project, helping inform decisions around project governance and how new contributors are welcomed.
+
+## **Want to Know More?**
+
+<span markdown="1"><details>
+<summary>Click to read more about this metric.</summary>
+
+### **Data Collection Strategies**
+- **Interviews or Surveys:** Ask community members to recognize others for their contributions to identify contribution types that may have been overlooked. Example questions:
+   - *Who in the project would you like to recognize for their contributions? What did they contribute?*
+- **Observations:** Identify and recognize leaders in different parts of the project.
+   - *What leaders are listed on the project website or in a repository?*
+- **Non-code Contributions Tracking:** Use a dedicated system, e.g., an issue tracker, to log non-code contributions.
+   - For example, communication channel activity like mailing list discussions can be a proxy for QA contributions.
+- **Trace Data Collection:** Measure contributions through collaboration tool logs (e.g., repository commits, wiki edits, or email archives).
+- **Automated Classification:** Train an AI bot to detect and classify contributions (e.g., distinguishing help requests from support provided).
+
+### **Filters**
+- Contribution types might include:
+  - Writing Code
+  - Reviewing Code
+  - Bug Triaging
+  - Quality Assurance and Testing
+  - Security-Related Activities
+  - Localization and Translation
+  - Event Organization
+  - Documentation Authorship
+  - Community Building and Management
+  - Social Media Management
+  - Marketing and Campaign Advocacy
+
```

Comment:
Could you please update the list to include all of the types of contributions that were in the original document.

Reply:
Done. Thank you Matt.